build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Build package artifacts as part of RPM/DEB build process.
+# Expected environment variables:
+#
+# For build action:
+#     GOVERSION: value like "go1.14.1" that specifies what version of Go
+#                will be downloaded and installed IF there is no Go found.
+# For install action:
+#     BUILDROOT: target directory for all packaged files (artifacts).
+
+set -e -x
+echo 'Going to cd into directory'
+cd $(dirname $0)
+
+ACTION="$1"
+REPO_ROOT=$(pwd)
+
+do_build() {
+        # ensure go is installed
+        if ! go version >& /dev/null; then
+                test -n "$GOVERSION"
```

Comment:
I strongly disagree to install golang in build scripts. Could you please different make target for that